EDSAnalytics/EDSTest/azure-pipeline.yml

```diff
@@ -0,0 +1,72 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - '/EDSAnalytics'
+
+pr:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - '/EDSAnalytics'
+
+schedules:
+  - cron: '0 0 * * 0'
+    displayName: Weekly build
+    branches:
+      include:
+        - master
+    always: true
+
+variables:
+  - template: '/miscellaneous/build_templates/variables.yml'
+  - name: projPath
+    value: '/EDSAnalytics/EDSTest'
+  - name: testPath
+    value: '/EDSAnalytics/EDSTestTest'
+  - name: CredFile
+    value: appsettings.json
+  - name: CredFileLibrary
+    value: product-readiness.sds_dotnet_rest.json
+  - name: covStream
+    value: SDSDotNetAPI
+  - name: bdProject
+    value: SDS_DotNet_REST
+
+jobs:
+  - job: Tests
+    strategy:
+      matrix:
+        linux:
+          imageName: 'ubuntu-16.04'
+        windows:
+          imageName: 'vs2017-win2016'
+    pool:
+      vmImage: $(imageName)
+    steps:
+      - template: '/miscellaneous/build_templates/config.yml'
+        parameters:
+          fileOriginal: $(CredFile)
+          fileLibrary: $(CredFileLibrary)
+          filePath: $(projPath)
+
+      - script: |
```

Comment:
There's actually a much cleaner way to run/test the app, see [this pipeline](https://github.com/osisoft/OSI-Samples-OCS/blob/master/basic_samples/Authentication/ClientCredentialFlow/DotNet/azure-pipeline.yml#L54) for an example